xcodeproj/internal/bazel_integration_files/generate_bazel_dependencies.sh

```diff
@@ -136,6 +136,12 @@ else
   readonly config="_${BAZEL_CONFIG}_build"
 fi
 
+# Respect the "Continue building after errors" setting
+continue_building_value="$(defaults read com.apple.dt.Xcode IDEBuildingContinueBuildingAfterErrors 2>/dev/null)"
+if [ "$continue_building_value" == "1" ]; then
+  build_pre_config_flags+=("--keep_going")
```

Comment:
What happens if someone is already setting this in their bazelrc?

Reply:
`--keep_going` / `--no_keep_going` seems to be one of those flags you can set as many times as you want (last in wins). Since this is set before the bazelrcs are applied, whatever is in the bazelrc should win.

Reply:
It overrides globals, but allows the `rules_xcodeproj` config to override it.

Reply:
Would it be possible then to set this Xcode setting the other way then? Right now the setting will only match if it's set in Xcode right?

Reply:
I could add an `else` with `--nokeep_going` if you'd prefer @luispadron

Reply:
Yeah, I think we should have it set both states, so it matches exactly what Xcode says, unless someone overrides it in the `rules_xcodeproj` config.